sys/conf/Makefile.arm64

```diff
@@ -27,6 +27,12 @@ S=	../../..
 
 INCLUDES+= -I$S/contrib/libfdt
 
+# TODO: This shouldn't be needed, we should be building with TARGET_ARCH=morello
```

Comment:
No to the second part for two reasons:
1. it's aarch64c now
2. that would be a purecap kernel

Should we not have TARGET_CPUTYPE=morello already? This really should not be needed, and does not belong in dev. This kind of thing is supposed to be set in kern.mk based on `.if ${MACHINE_CPU:Mcheri}` (see RISC-V).